### PR TITLE
Upgrade dapp deps

### LINKED ISSUE #127: Update dApps to new web-sdk and wallet-api

### PR DESCRIPTION
**Pull request by soerenbf**
## Purpose

Update dapps to new web-sdk and align build tools.

Kept as draft until libraries are officially released

Closes #127

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

**Review by abizjak — COMMENTED** [review]
Why is it still in draft?

**Comment by soerenbf** [comment]
> Why is it still in draft?

To keep it unmerged until the next wallet version has been released.

File: test-tools/proof-explorer/src/Root.tsx [hunk 1]
```diff
@@ -187,14 +190,19 @@ interface RevealAttributeProps {
...
     setMessages: (updateMessage: (oldMessages: string[]) => string[]) => void
 ) {
     let proof: VerifiablePresentation;
     const challengeBuffer = new Uint8Array(32);
     crypto.getRandomValues(challengeBuffer);
     const challenge = Buffer.from(challengeBuffer).toString('hex');
     console.log(statement);
+
+    if (provider === undefined) {
+        return;
```
[the remark this answers is not in the document]
Comment (soerenbf) [reply test-tools/proof-explorer/src/Root.tsx]: Went with a better alternative

File: examples/some-verifier/frontend/src/components/Issuer.tsx [hunk 2]
```diff
@@ -273,9 +274,10 @@ async function requestCredential(
   // eslint-disable-next-line no-constant-condition
   while (true) {
     try {
-      const itemSummary = await api
-        .getGrpcClient()
-        .waitForTransactionFinalization(txHash!);
+      const client = new ConcordiumGRPCClient(api.grpcTransport);
```
Comment (abizjak) [inline examples/some-verifier/frontend/src/components/Issuer.tsx:277]: This loop is still constructing a client per iteration.

**Review by abizjak — APPROVED** [review]
Could you please bump versions of all the components that have been updated?

And reflect the updates in the changelog.

File: examples/some-verifier/frontend/src/components/Issuer.tsx (hunk 2 above)
Comment (soerenbf) [reply examples/some-verifier/frontend/src/components/Issuer.tsx]: Well it appears there were 2 instances of this, so it's not the same one as the one referenced by your previous comment 😛

**Pull request merged**